aws/s3/s3.go

```diff
@@ -243,7 +243,7 @@ func (s *S3) GetContentSizeTime(bucket, key string) (int64, time.Time, error) {
 	if err != nil {
 		return size, mt, err
 	}
-	return o.ContentLength, *o.LastModified, nil
+	return *o.ContentLength, *o.LastModified, nil
```

Comment:
should probably use `aws.ToString()` / `aws.String()` (can't remember which in this case). Is safer if pointer is nil.

Probably review all the pointers in this PR.

Reply:
This is AWS SDK changed the member type in the returned structure from int64 to *int64, not the calling parameters.

And the reason I didn't do pointer check is because the `*o.LastModified` has been living well, also there's already a error check right above, and apparently the ContentLength is not an optional attribute. So I think it should be fine to just de-referencing it.

Reply:
Also for the `IsTruncated` below, I don't think it's optional tool. It must have zero value when the sdk forms the structure.